### PR TITLE
Added ability to change stroke settings in config.php and config.defa…

### PR DESCRIPTION
**Pull request by captainpovey**
Made a few simple changes to give ability to change the stroke Weight, Colour and Opacity from the config.php file. This partially addresses #64
I've added three variables at the bottom of config.default.php as I obviously don't want to upload config.php!
This is my first time using github - I've tried to be careful!
I have some other changes I've worked on - specifically, ability to delete a point from the pop-up
I'm also working on being able to select and show points on a given date on a given track.
Not sure if you are still maintaining this project!

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -641,8 +641,13 @@ function htmlEncode(s) {
           .replace(/>/g, '&gt;');
 }
 
+// Convert hex string and opacity to an rgba string
+function hexToRGBA(hex, opacity) {
+	return 'rgba(' + (hex = hex.replace('#', '')).match(new RegExp('(.{' + hex.length/3 + '})', 'g')).map(function(l) { return parseInt(hex.length%2 ? l+l : l, 16) }).concat(opacity||1).join(',') + ')';
```
Comment (bfabiszewski) [inline js/main.js]: Could you please change indentation here to 2 spaces?

Comment (captainpovey) [reply js/main.js]: Hi. Yes I will. I'm also addressing the failed check. Cheers.

Comment (bfabiszewski) [reply js/main.js]: The problems reported by codacy seem to be false positives

File: js/api_openlayers3.js [hunk 2]
```diff
@@ -73,8 +73,8 @@ function init() {
   // init layers
   var lineStyle = new ol.style.Style({
     stroke: new ol.style.Stroke({
-      color: 'rgba(255, 0, 0, 0.5)',
-      width: 2
+      color: hexToRGBA(strokeColor, strokeOpacity),
```
Comment (bfabiszewski) [inline js/api_openlayers3.js]: One more thing. The function hexToRGBA is not needed. The color may also be given in hex format. See [openlayers color](https://openlayers.org/en/latest/apidoc/ol.color.html).

**Comment by captainpovey** [comment]
Changed to 2 spaces. 
I did look into things regarding rgba / hex - but I read that there is no opacity style in openlayersv4.6.x

**Comment by bfabiszewski** [comment]
Thanks for the correction.
Regarding hex to rgba it was my mistake, I didn't notice opacity is needed, sorry!

If you want to contribute the other features, you are welcome. I wanted to add the ability to delete a point from the pop-up long time ago. But I didn't.

**Comment by captainpovey** [comment]
No problem! 
I'll work on tidying the 'delete point' code.I've found it very useful to correct some wayward gps readings.
If you get chance to merge then that would be cool - thanks for creating the code in the first place. I'm hoping it'll allow me to break with google location history one day.

**Pull request merged**